github-repo-manager/src/main/java/com/axonivy/github/GitHubFileDetector.java

```diff
@@ -0,0 +1,110 @@
+package com.axonivy.github;
```

Comment:
Since all your new classes and GitHubMissingLicenseDetector belong together, I would put them into a new package com.axonivy.github.file